lib.rs

```diff
@@ -1157,10 +1158,25 @@ where A::Item: Deserialize<'de>,
 }
 
 impl<'a, A: Array> From<&'a [A::Item]> for SmallVec<A> where A::Item: Clone {
+    #[cfg(not(feature = "specialization"))]
     #[inline]
     fn from(slice: &'a [A::Item]) -> SmallVec<A> {
         slice.into_iter().cloned().collect()
     }
+
+    #[cfg(feature = "specialization")]
+    #[inline]
+    default fn from(slice: &'a [A::Item]) -> SmallVec<A> {
```

Comment:
Doesn't that mean that one can specialise the implementation from outside the crate? Shouldn't we be specialising an internal trait used by this impl?

Reply:
I'll add an internal SpecFrom trait then.

Reply:
Done.